package.json

```diff
@@ -55,7 +55,8 @@
     "ua-parser-js": "^0.7.18",
     "underscore.string": "^3.3.4",
     "unfetch": "^3.0.0",
-    "wicg-focus-ring": "https://github.com/dakeyras7/focus-ring#include-enter-keydown"
+    "wicg-focus-ring": "https://github.com/dakeyras7/focus-ring#include-enter-keydown",
+    "cordova-android": "~7.0.0"
```

Comment:
Is this a non-dev dependency or can we move it to all the other cordova things?

Reply:
yarn.lock was not updated, though, was this intentional?

Reply:
You are right. This was added automatically when building for android via `yarn build-android` I guess.

Reply:
@opyh mentioned that cordova tends to reorder the package.json :/

Reply:
🙄 I will remove it.